src/index.js

```diff
@@ -1,8 +1,12 @@
-const contentDiv = document.querySelector("#content");
-const navLinks = document.querySelectorAll(".nav-link");
+import { Home } from './homepage';
+import { Contact } from './contactpage';
+import { Menu } from './menupage';
+
+const contentDiv = document.querySelector('#content');
+const navLinks = document.querySelectorAll('.nav-link');
 const homeContent = `<div class="container" id="home-section">
```

Comment:
Could you add this to the homepage module instead? Kindly do the same for other modules.